.github/workflows/python-package.yml

```
@@ -24,14 +23,27 @@ jobs:
         run: |
           pip install setuptools wheel twine build
           python -m build
-      - name: Upload build artifact
+      - name: Upload tar artifact
         uses: actions/upload-artifact@v3.1.0
         with:
           name: arelle.tar.gz
           path: dist/*.tar.gz
+      - name: Upload wheel artifact
+        uses: actions/upload-artifact@v3.1.0
+        with:
+          name: arelle.whl
+          path: dist/*.whl
+
+  publish-package:
+    if: startsWith(github.ref, 'refs/tags')
+    needs: build-package
+    environment: release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download artifacts
+        uses: actions/download-artifact@v3
```

Use the full action version and let dependabot manage updates:
```suggestion
        uses: actions/download-artifact@v3.0.0
```